journalism_workflow/static/journalism_workflow/photographer/partials/photographer.html

```diff
@@ -0,0 +1,80 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Take and upload photos for the article
+        </h3>
+      </div>
+    </div>
+
+    <!-- Titles for the two columns -->
+    <div class="row padded">
+      <div class="col-lg4 col-md-4 col-sm-4">
+	<h3>Editor's Story Plan</h3>
+      </div>
+
+      <div class="col-lg-8 col-md-8 col-sm-8">
+	<h3>Photographer Instructions</h3>
+      </div>
+
+    </div>
+
+    <div class="row padded">
+
+      <!-- Form with Editor's story plan -->
```

Comment:
if @derek-schultz or @jrbotros clean up angular organization, this won't be swept up in it---putting the editor form into a directive will just mean that you can reuse it in the two places it's copy/pasted now.

that said, if we want to create a global "improve the journalism workflow example" issue and add this to the list, i'm ok with that